app/src/main/res/layout/merge_box.xml

```diff
@@ -8,7 +8,7 @@
         android:layout_height="250dp"
         android:layout_centerVertical="true"
         android:text="SWITCH TO INCLUDE"
-        android:onClick="click2"/>
+        android:onClick="click"/>
```

Comment:
method names should be relevant.
Moreover on a side note you can use the same method also for different view, as it s definition contains a object of the view on which onClick is called in the parameters.